src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Ball;
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
```

Comment:
The getRandomBall() method should not be a static
ColorSupplier and Random must be class fields